.github/workflows/ci.yml

```
@@ -97,6 +97,22 @@ jobs:
             }}
           token: ${{ secrets.CODECOV_TOKEN }}
 
+      - name: Upload awx jUnit test reports
+        if: >-
+          matrix.test.name == 'api-test'
+          && steps.make-run.outputs.test-result-files != ''
+          && github.event_name == 'push'
+          && github.repository == 'ansible/awx'
```

This is more reliable and how I've implemented the upstream check above.
```suggestion
          && env.UPSTREAM_REPOSITORY_ID == github.repository_id
```